integration/volume_test.go

```diff
@@ -75,3 +75,23 @@ func (s *CliSuite) TestNamedVolume(c *C) {
 	c.Assert(cn.Mounts[0].Name, DeepEquals, "vol")
 	c.Assert(cn.Mounts[0].Destination, DeepEquals, "/path")
 }
+
+func (s *CliSuite) TestV2Volume(c *C) {
```

Comment:
I shoud complete this one to make sure `Bind` and/or `Mounts` uses `project_test` as the name for the volume 👼